iOS/Flows/Home/Moments/Capture/PIP/PiPRecordingViewController.swift

```diff
@@ -135,9 +136,15 @@ class PiPRecordingViewController: ViewController, AVCaptureVideoDataOutputSample
     // MARK: - PUBLIC
     
     func startRecording() {
-        let settings = self.backOutput.recommendedVideoSettingsForAssetWriter(writingTo: .mp4)
-        let audioSettings = self.micDataOutput.recommendedAudioSettingsForAssetWriter(writingTo: .mp4)
-        self.recorder.initialize(backVideoSettings: settings, audioSettings: audioSettings)
+        var backVideoSettings = self.backOutput.recommendedVideoSettingsForAssetWriter(writingTo: .mov)
+        // Adjust the bitrate for change video size (less bitrate, less size and quality)
+        // This bitrate will need to be adjusted for videos that are higher res than 1080.
+        var compressionSettings = backVideoSettings?[AVVideoCompressionPropertiesKey] as! [String : Any]
+        compressionSettings[AVVideoAverageBitRateKey] = NSNumber(integerLiteral: 3_500_000)
+        backVideoSettings?[AVVideoCompressionPropertiesKey] = compressionSettings
```

Comment:
I take the recommended settings, and then adjust the bitrate down. The bitrate is how much data per second, so the lower the bitrate, the greater the compression.

The specific number used here is just my best attempt at balancing size and quality.

For reference, the recommended bitrate was about 7_000_000 on my device, an iPhone 12 mini.